backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -20,6 +20,16 @@ class SampleRowHandler(GenericRowHandler):
     def __init__(self):
         super().__init__()
 
+    def validate_row_input(self, **kwargs):
+        super().validate_row_input(**kwargs)
+
+        sample = kwargs.get("sample", None)
```

Comment:
Why not assume that the expected parameters are present? `sample, library, container, project, parent_container, individual, sample_kind_objects_by_name, defined_pools`
```
    def process_row_inner(self, sample, library, container, project, parent_container, individual, sample_kind_objects_by_name, defined_pools):
```

Reply:
The validate row input signature would be different for a different template row ...